06a-WGS.Rmd

```diff
@@ -0,0 +1,118 @@
+ottrpal::include_slide("https://docs.google.com/presentation/d/1YwxXy2rnUgbx_7B7ENH9wpDX-j6JpJz6lGVzOkjo0qY/edit#slide=id.g138a6ce16b7_35_43")
+```
+The following link provides the data analysis pipeline written by researchers in the NCI division of the NIH and provides a helpful overview of the typical steps necessary for [WGS analysis](https://docs.gdc.cancer.gov/Data/Bioinformatics_Pipelines/DNA_Seq_Variant_Calling_Pipeline/).
+
+Here are many of the tools and resources used by researchers for analyzing WGS data.
+First, these are common tools for data pre-processing:
+
+- [Bedtools](https://bedtools.readthedocs.io/en/latest/) including the bamtofastq function, which is the first step in converting data off the sequencer to a useable format for downstream analysis
+- [Samtools](http://www.htslib.org/) including tools for converting fastq to BAM files while mapping genes to the genome, duplicate read marking, and sorting reads
+- [Picard2](https://broadinstitute.github.io/picard/) including tools to covert fastq to SAM files, filter files, create indices, mark read duplicates, sort files, and merge files
+- [GATK](https://gatk.broadinstitute.org/hc/en-us) is a comprehensive set of tools from the Broad Institute for analyzing many types of sequencing data. For pre-processing, the print read function is very beneficial for writing the reads from a BAM or SAM file that pass specific criteria to a new file
+
+These are the common tools used for somatic and germline variant identification. Note many researchers will often use a combination of these tools to narrow down only variants that are identified using a combination of these analysis algorithms:
+- [Mutect2](https://gatk.broadinstitute.org/hc/en-us/articles/9570422171291-Mutect2) This is a beneficial variant calling tool with functions including using a "panel of normals" to better compare disease samples to normal and filtering functions for samples with orientation bias artifacts (FFPE samples) called F1R2
+- [Varscan 2](https://varscan.sourceforge.net/) This is a helpful tool that utilizes a heuristic/statistic approach to variant calling. It is generally used and has capabilities for calling multi-sample variants. It can also be used for identifying copy number alterations in tumor-normal pairs
+- [MuSE](https://bioinformatics.mdanderson.org/public-software/muse/) This is a beneficial mutation calling tool when you have both tumor and normal datasets. It uses a Markov Substitution model for Evolution, which models the evolution of the allelic composition of the tumor and normal tissue at each reference base. 
+- [SvABA](https://github.com/walaj/svaba) This tool is especially useful for calling insertions and deletions (indels) because it utilizes a String Graph Assembler to assemble aberrantly aligned sequence reads that reflect indels or structural variants. Indels can be difficult to detect with standard alignment-based variant callers. 
```

Comment:
```suggestion
- [SvABA](https://github.com/walaj/svaba) This tool is especially useful for calling insertions and deletions (indels) because it utilizes a String Graph Assembler to assemble aberrantly aligned sequence reads that reflect indels or structural variants. Indels can be difficult to detect with standard alignment-based variant callers. 
```
I love the sentence about Indels, because that really tells me why I might want to use SvABA -- aka if I'm interested in indels. 
I have similar comment here about `String Graph Assembler` what does this mean to me as a user? And can we link to more info?